_posts/2020-08-10-release-2.11.0.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: Release v2.11.0
+tags: ["v2.11.0", "changelog"]
+excerpt_separator: <!--more-->
+---
+
+As of 10/08 2020 the new version 2.11.0 is out
+<!--more-->
+
+## Standardization and Conformance Improvements
+
+- [#1371]({{ site.github.issues_url }}/1371) It's now possible to run Standardization and Conformance at once instead of one after each other. This is suitable when their resource requirements are rather similar and it saves time by initializing only once
```

Comment:
```suggestion
- [#1371]({{ site.github.issues_url }}/1371) It's now possible to run Standardization and Conformance as one job instead of one after each other. This is suitable when their resource requirements are rather similar and it saves time by initializing spark session only once
```